docs/host/host.md

```diff
@@ -21,6 +21,28 @@ Because Harvester is built on top of Kubernetes and uses etcd as its database, t
 
 ## Node Maintenance
 
+
+:::warning
+
+We found that there is a chance that VMs on a node can encounter an I/O error while the node enters maintenance mode. Please follow this procedure to avoid the problem before a fix is released:
```

Comment:
```suggestion
A recent [bug](https://github.com/harvester/harvester/issues/7128) reveals that there is a chance that VMs on a node can encounter an I/O error while in maintenance mode. Users [must|should] follow the following maintenance procedure until a fix is released:
```
I think if this is something that we want every user to follow, we should be more explicit about it.